virttest/http_server.py

```diff
@@ -69,7 +69,7 @@ def send_head_range(self, range_begin, range_end):
             # Always read in binary mode. Opening files in text mode may cause
             # newline translations, making the actual size of the content
             # transmitted *less* than the content-length!
```

Comment:
it need special care here,see above comments

Reply:
Accepted, will not change to text